funcx_endpoint/tests/funcx_endpoint/executors/high_throughput/test_funcx_manager.py

```diff
@@ -28,35 +33,25 @@ def test_remove_worker_init(self):
         assert task.task_id == "KILL"
         assert task.task_buffer == "KILL"
 
-    def test_manager_worker(self):
+    def test_poll_funcx_task_socket(self, mocker):
+        mocker.patch('funcx_endpoint.executors.high_throughput.funcx_manager.zmq.Context')
```

Comment:
Add a note here about why you are mocking ZMQ. We need some info that ZMQ in tests are causing test hangs.